crates/filament/src/cmdline.rs

```diff
@@ -104,9 +100,9 @@ pub struct Opts {
     /// backend to use (default: verilog): calyx, verilog
     #[argh(option, long = "backend", default = "Backend::Verilog")]
     pub backend: Backend,
-    /// disable generation of slow FSMs in the backend
-    #[argh(switch, long = "disable-slow-fsms")]
-    pub disable_slow_fsms: bool,
+    /// disable generation of counter FSMs in the backend
```

Comment:
What is a counter FSM? Or rather, how is a new user supposed to know what a counter FSM is?